### PR TITLE
database up and running migrations work shopping cart table has dummy…

### PR DESCRIPTION
**Pull request by BrianM-218**
… seed data in it

### CODE REVIEW

**Review by desicurry — APPROVED** [review]
Everything looks good. waiting for one more review to merge

**Comment by Air-Bear** [comment]
Well done on adding the framework for the buyer cart functionality and wrapping up those loose bugs. The code looks good as far as I can tell so I'm moving forward with the merge.

**Pull request merged**
